lambda_datasci_perf/function/timing.py

```diff
@@ -0,0 +1,13 @@
+import time
+
+class Timer:
+
+    def __init__(self):
+        self.time = 0
+        self.elapsed_us = 0
+
+    def __enter__(self):
+        self.time = time.time_ns()
```

Comment:
If you want you can use time.percounter_ns() for a guaranteed highest resolution performance counter (in nanoseconds) or time.monotonic_ns() for a guaranteed monotonically increasing time (in nanoseconds). This prevents any time zone related funnies kicking in.

https://docs.python.org/3/library/time.html#time.perf_counter_ns

https://docs.python.org/3/library/time.html#time.monotonic_ns